processor/tests/src/main/java/org/mobilitydata/gtfsvalidator/processor/tests/ManyFieldsSchema.java

```diff
@@ -0,0 +1,26 @@
+package org.mobilitydata.gtfsvalidator.processor.tests;
+
+import org.mobilitydata.gtfsvalidator.annotation.GtfsTable;
+
+@GtfsTable("many_fields.txt")
+public interface ManyFieldsSchema {
```

Comment:
Do you think it's worth testing an edge case when the number of fields is equal to the number of bits and the last field is handled correctly?

Reply:
Changed to 8 fields.